### PR TITLE
feat: blog page start

### PR DESCRIPTION
**Pull request by moripen**

### CODE REVIEW

File: src/pages/blog.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    maxWidth: 'container.xl',
+    borderRadius: '2xl'
+  },
+  shouldForwardProp: prop =>
+    ['width', 'height', 'src', 'alt', 'fill', 'quality'].includes(prop),
+});
+
+const Blog: NextPage = () => (
+  <Layout>
+    <Container p="l">
```
Comment (cobraz) [inline src/pages/blog.tsx:22]:
```suggestion
    <Container>
```

I don't think `l` makes sense here. Did you check mobile view?

File: src/pages/blog.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      <Hero
+        src={veltBoss}
+        alt="Picture of the boss in Velt"
+        quality={100}
+        placeholder="blur"
+      />
+      <Flex flexDirection="column" pt="m" gap="l">
```
Comment (cobraz) [inline src/pages/blog.tsx:29]: I'm not sure we need this. Seeing as all the child `<Flex />`'es are setting some form of padding value. We're using this only to achieve the `gap` between elements, right?

**Comment by braaar** [comment]
The margins on the side are too large on mobile

**Review by cobraz — APPROVED** [review]
We're not going to use these images in the final product; we'll pull them from Sanity.

Don't worry too much about images missing in the preview. AFAIK, they work on standard deployments.

**Review by braaar — APPROVED** [review]
In general, this looks like a fine first draft. Setting up a blog post component that takes in data instead of being hard coded will come in a later PR, I assume.

File: src/pages/blog.tsx (hunk 1 above)
Comment (cobraz) [reply src/pages/blog.tsx:22]: See @braaar's comment: https://github.com/veltolini/website/pull/34#issuecomment-1459637291

File: src/pages/blog.tsx (hunk 2 above)
Comment (moripen) [reply src/pages/blog.tsx:29]: Ah, yeah we have the Container around it regardless. The idea was to have it all in a Flex to make the transition to being CRM-based later easier, but I guess the Container does the trick.

**Pull request merged**